examples/AffinePlaneCurve.jl

```diff
@@ -0,0 +1,124 @@
+module AffinePlaneCurveModule
+
+using Oscar, Markdown
+
+export AffinePlaneCurve, AffinePlaneCurve_degree, AffinePlaneCurve_Jacobian, AffinePlaneCurve_tangent, AffinePlaneCurve_issmoothpoint, AffinePlaneCurve_union
+
+####################################################
+#
+#We follow the lecture notes by Andreas Gathmann: https://www.mathematik.uni-kl.de/~gathmann/en/curves.php
```

Comment:
This would look nicer if there was a space before the "We" and if the URL was on a new line and there were 80 hashes (the console width in many consoles.